src/event-db/src/types/vit_ss/vote_plan.rs

```diff
@@ -0,0 +1,59 @@
+use crate::types::utils::serialize_datetime_as_rfc3339;
```

Comment:
Are you using the views defined in `V9__vitss_compatibility.sql`??

Reply:
No, I have not relied on `V9__vitss_compatibility.sql`, I have added brand new queries. Of course they are mostly the same what we can see inside `V9__vitss_compatibility.sql`, but it is a separate queries.